structured_data/2022_06_02_causality/src/transfer_entropy/README.md

```diff
@@ -0,0 +1,5 @@
+## Purpose of this folder
+We advocate to use the [PyCausality](https://pypi.org/project/PyCausality/) package when working with Transfer Entropy. 
+The code itself is not maintained anymore but still works well. 
+Because the pip package might become/is unstable, we copied the source code and created a wrapper around it.
```

Comment:
Is the pip package actually unstable? Or is the main reason here that our wrapper adds a lot of value?